src/components/tiles/geometry/geometry-content.tsx

```diff
@@ -56,6 +56,7 @@ import ErrorAlert from "../../utilities/error-alert";
 import { halfPi, normalizeAngle, Point } from "../../../utilities/math-utils";
 import SingleStringDialog from "../../utilities/single-string-dialog";
 import { getClipboardContent, pasteClipboardImage } from "../../../utilities/clipboard-utils";
+import { ToolTitleArea } from "../tile-title-area";
```

Comment:
Good point. I'll make it part of the next PR. I don't want to update this one right now since it would take another 100 minutes to prove that nothing was broken.